main.tf

```diff
@@ -39,6 +39,8 @@ resource "aws_security_group_rule" "https_ingress" {
 }
 
 module "access_logs" {
+  count = var.access_logs_s3_bucket_id == "" ? 1 : 0
```

Comment:
GitHub doesn't allow me to comment on line 83 of this file, but that will have to be changed from:
```
bucket = module.access_logs.bucket_id
```
To:
```
bucket = coalesce(var.access_logs_s3_bucket_id, module.access_logs.bucket_id)
```